WalletWasabi.Fluent/ViewModels/Settings/GeneralSettingsTabViewModel.cs

```diff
@@ -55,22 +57,20 @@ public GeneralSettingsTabViewModel()
 				.Skip(1)
 				.Subscribe(x => Services.UiConfig.Autocopy = x);
 
-			this.WhenAnyValue(x => x.RunOnSystemStartup)
-				.ObserveOn(RxApp.MainThreadScheduler)
-				.Skip(1)
-				.Subscribe(async runOnStartup =>
+			StartUpCommand = ReactiveCommand.Create(async () =>
+			{
+				try
 				{
-					try
-					{
-						await StartupHelper.ModifyStartupSettingAsync(runOnStartup);
-						Services.UiConfig.RunOnSystemStartup = runOnStartup;
-					}
-					catch (Exception ex)
-					{
-						Logger.LogError(ex);
-						await ShowErrorAsync(Title, "Couldn't save your change, please see the logs for further information.", "Error occured.");
-					}
-				});
+					await StartupHelper.ModifyStartupSettingAsync(RunOnSystemStartup);
+					Services.UiConfig.RunOnSystemStartup = RunOnSystemStartup;
+				}
+				catch (Exception ex)
+				{
+					Logger.LogError(ex);
+					RunOnSystemStartup = !RunOnSystemStartup;
+					await ShowErrorAsync(Title, "Couldn't save your change, please see the logs for further information.", "Error occured.");
```

Comment:
```suggestion
					await ShowErrorAsync(Title, "Couldn't save your change, please see the logs for further information.", "Error occurred.");
```
(https://www.thefreedictionary.com/occurred imho)